packages/react/src/components/header/components/headerActionBar/HeaderActionBar.tsx

```diff
@@ -89,16 +152,37 @@ export const HeaderActionBar = ({
   logo,
   onTitleClick,
   onLogoClick,
-  onMenuButtonClick,
   children,
   className,
   ariaLabel,
   role,
+  frontPageLabel,
+  openFrontPageLinksAriaLabel,
 }: HeaderActionBarProps) => {
   const handleClick = useCallbackIfDefined(onTitleClick);
   const handleLogoClick = useCallbackIfDefined(onLogoClick);
   const handleKeyPress = useEnterOrSpacePressCallback(handleClick);
   const handleLogoKeyPress = useEnterOrSpacePressCallback(handleLogoClick);
+  const { mobileMenuOpen } = useHeaderContext();
+  const actionBarRef = useRef<HTMLDivElement>();
+
+  useEffect(() => {
+    // When mobile menu is open, set up tab barriers to prevent keyboard navigation to content outside action bar and menu.
+    if (mobileMenuOpen && actionBarRef !== undefined) {
+      addDocumentTabBarrier(TabBarrierPosition.top, actionBarRef.current);
+      addDocumentTabBarrier(TabBarrierPosition.bottom, actionBarRef.current);
```

Comment:
It would be cleaner if `addDocumentTabBarrier` returns a function to remove the created element. 

Current solution is ok, but for example if className in `addDocumentTabBarrier` is changed, then barriers wont find elements.

At least the "hds-actionbar-tab-barrier" should be a constant to prevent this.

Reply:
https://github.com/City-of-Helsinki/helsinki-design-system/pull/1120/commits/31a41d6981488d00f94541b41f1caff492e3bacc